index.html

```diff
@@ -23,22 +23,18 @@
     <h1>Welcome to the student showcase!</h1>
   </div>
   <div class="content">
-    <div id="Tony">
-      <div class="container">
-        <div class="header">
-          <h1>Tony Bush</h1>
-        </div>
-        <div class="qoute">
-          <h2>Aint no thing but a chicken wing!</h2>
-        </div>
-      </div>
-      <div class="contact">
-        <ul>
-          <li><a href="https://github.com/Tonail">GitHub</a></li>
-          <li><a href="mailto:tbush01@gmail.com">EMAIL</a></li>
-          <li><a href="https://www.behance.net/tbush011874">Behence</a></li>
-        </ul>
-      </div>
+    <div class="Tony">
```

Comment:
These changes are appearing because of a deviation between your history and production. Let's work together to merge these changes before/during/after the next class, if we can.